lib/cylc/task_proxy.py

```diff
@@ -893,6 +1117,48 @@ def set_from_rtconfig(self, cfg=None):
             copy(GLOBAL_CFG.get(['execution polling intervals'])),
             'execution', self.log)
 
+    def register_job_logs(self, submit_num, result=None):
+        """Register job logs in the runtime database."""
+        data = []
+        if result is None:
+            job_log_dir = self.get_job_log_dir(
+                self.suite_name, self.tdef.name, self.point, submit_num)
+            try:
+                for filename in os.listdir(job_log_dir):
+                    if filename.startswith("."):  # ignore hidden files
```

Comment:
This one is hard to argue for or against! Let me know your preference, and I'll go for that.